### PR TITLE
Add python3-networkmanager to rosdep/python.yaml

### PR DESCRIPTION
**Pull request by furushchev**
Please add the following dependency to the rosdep database.

## Package name:

python3-networkmanager

## Package Upstream Source:

https://github.com/seveas/python-networkmanager

## Purpose of using this:

python3 version of `python-networkmanager`
https://github.com/ros/rosdistro/blob/master/rosdep/python.yaml#L2815-L2820

## Links to Distribution Packages

- Debian: https://packages.debian.org/
  - https://packages.debian.org/buster/python3-networkmanager
- Ubuntu: https://packages.ubuntu.com/
   - https://packages.ubuntu.com/focal/python3-networkmanager
- Fedora: https://src.fedoraproject.org/browse/projects/
  - https://src.fedoraproject.org/rpms/python-networkmanager#bodhi_updates
- Arch: https://www.archlinux.org/packages/
  - N/A
- Gentoo: https://packages.gentoo.org/
  - N/A
- macOS: https://formulae.brew.sh/
  - N/A
- Alpine: https://pkgs.alpinelinux.org/packages
  - N/A
- NixOS/nixpkgs: https://search.nixos.org/packages
  - N/A

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -6729,6 +6729,12 @@ python3-netifaces:
   openembedded: [python3-netifaces@meta-ros-common]
   rhel: ['python%{python3_pkgversion}-netifaces']
   ubuntu: [python3-netifaces]
+python3-networkmanager:
+  debian:
+    buster: [python3-networkmanager]
+    jessie: [python3-networkmanager]
+    stretch: [python3-networkmanager]
+  ubuntu: [python3-networkmanager]
```
Comment (cottsay) [inline rosdep/python.yaml:6737]: The automated checks have suggested a change that looks correct:
> Key 'python3-networkmanager' might be satisifed by fedora package named 'python3-networkmanager': https://src.fedoraproject.org/rpms/python-networkmanager#bodhi_updates

```suggestion
  fedora: [python3-networkmanager]
  ubuntu: [python3-networkmanager]
```

**Review by cottsay — DISMISSED** [review]

File: rosdep/python.yaml (hunk 1 above)
Comment (furushchev) [reply rosdep/python.yaml:6737]: @cottsay Thank you for the review!
Just a double check. The link refers to `python-networkmanager` ( not a `python3-networkmanager` ). Is it still ok to commit the suggestion?
(I'm not a user of fedora and I'm not sure the name convention of fedora packages.)

Comment (sloretz) [reply rosdep/python.yaml:6737]: @furushchev I think there is one source rpm `python-networkmanager`, but it is used to create a [`noarch` RPM called `python3-networkmanager`](https://koji.fedoraproject.org/koji/buildinfo?buildID=1754885). The `noarch` one is what's used to install the software. (@cottsay please correct me if I'm wrong)

Comment (cottsay) [reply rosdep/python.yaml:6737]: This is correct. The binary package name is `python3-networkmanager`. Thanks for checking.

Comment (furushchev) [reply rosdep/python.yaml:6738]: @cottsay @sloretz Thank you for double checking! I updated the commit.

**Comment by furushchev** [comment]
Test failed:

```
nosetests -s
  shell: /usr/bin/bash -e {0}
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.8.10/x64
Package 'python3-networkmanager' for debian buster on amd64 was found: https://packages.debian.org/buster/python3-networkmanager

Error: Package 'python3-networkmanager' could not be found for debian stretch on amd64
```

I removed a key for debian/stretch.

**Review by jacobperron — APPROVED** [review]

**Review by nuclearsandwich — APPROVED** [review]

**Pull request merged**